Runtime/Reporters/SlackMessageSender.cs

```diff
@@ -92,7 +92,7 @@ public SlackMessageSender(SlackAPI api)
                 return;
             }
 
```

Comment:
I think we should warn here if users specify withScreenshot on batchmode instead of just ignoring. Would you warn it?

Reply:
Isn't it the behavior before the change that requires a warn?
After changes, can running specify `withScreenshot` on batchmode.

Reply:
Ahhhhh, I'm sorry, I misunderstood. I got confused between before and after...